src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/finalclass/InputFinalClassNestedInRecord.java

```diff
@@ -0,0 +1,49 @@
+/*
+FinalClass
+
+
+*/
+
+//non-compiled with javac: Compilable with Java16
```

Comment:
We generally use the lowest language level that we can here, is there some syntax that prevents us from using 14?

Reply:
Yes, it is compilable with java 14, I usually used static declarations in inner classes in these types of examples which are only compilable with 16+, didn't use it in this one, changed it to 14.

Reply:
Resolved.